cmd/integration/integration.go

```diff
@@ -963,7 +963,7 @@ func main() {
 
 	go func() {
 		defer util.FlushLogs()
-		time.Sleep(3 * time.Minute)
+		time.Sleep(10 * time.Minute)
```

Comment:
nit: Ideally a named constant at the head of the file, rather than an undocumented literal buried in the code.
Also, what percentage of integration test runs have you actually seen timing out at 3 minutes?